common/mariadb/Chart.yaml

```
@@ -2,3 +2,4 @@ apiVersion: v1
 description: A Helm chart for Kubernetes
 name: mariadb
 version: 0.1.5
+version: 0.1.6
```

```suggestion
version: 0.1.6
```
I think that is what you meant